xpdan/pipelines/main.py

```diff
@@ -49,8 +35,8 @@ def conf_main_pipeline(db, save_dir, *, write_to_disk=False, vis=True,
                        mask_kwargs=None,
                        pdf_config=None,
                        verbose=False,
-                       calibration_md_path='../xpdConfig/'
-                                           'xpdAcq_calib_info.yml'):
+                       calibration_md_folder='../xpdConfig/'
+                       ):
```

Comment:
Correct folder should be ``xpdUser/config_base/``

Reply:
I'll load it in from the ipython profile.